mindjunkies/forums/models.py

```
@@ -117,7 +124,10 @@ class Reply(models.Model):
     created = models.DateTimeField(auto_now_add=True)
 
     def __str__(self):
-        return f"{self.author.username} : {self.body[:30]}"
+        try:
+            return f"{self.author.username} : {self.body[:30]}"
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)